### PR TITLE
Allow php8

### PR DESCRIPTION
**Pull request by odolbeau**

### CODE REVIEW

**Review by jderusse — COMMENTED** [review]
travis file should be updated IMHO

File: composer.json [hunk 1]
```diff
@@ -12,7 +12,7 @@
         "sort-packages": true
     },
     "require": {
-        "php": "^7.2",
+        "php": "^7.2|^8.0",
```
Comment (maxhelias) [inline composer.json:15]: Maybe we can use `>=7.2.5` like Symfony recommand ?

File: .travis.yml [hunk 2]
```diff
@@ -6,6 +6,7 @@ php:
   - 7.2
   - 7.3
   - 7.4
+  - 8.0
```
Comment (odolbeau) [inline .travis.yml:9]: @jderusse PHP 8.0 added! :ok_hand:

**Review by maxhelias — APPROVED** [review]

**Review by jderusse — APPROVED** [review]

**Comment by Ousret** [comment]
Hi, @odolbeau , could you create a new tag? thank you.

**Pull request merged**